jenkins_job_wrecker/modules/builders.py

```diff
@@ -102,3 +102,20 @@ def batchfile(child, parent):
                                       "XML %s" % shell_element.tag)
 
     parent.append({'batch': shell})
+
+
+def buildnameupdater(child, parent):
+    build_name = {}
+    for build_name_element in child:
+        if build_name_element.tag == "buildName":
+            build_name["name"] = build_name_element.text
+        if build_name_element.tag == "macroTemplate":
+            build_name["template"] = build_name_element.text
+        if build_name_element.tag == "fromFile":
+            build_name["file"] = (build_name_element.text == 'true')
+        if build_name_element.tag == "fromMacro":
+            build_name["macro"] = (build_name_element.text == 'true')
+        if build_name_element.tag == "macroFirst":
+            build_name["macro-first"] = (build_name_element.text == 'true')
```

Comment:
Please change the above "if"s to "elif"s and add an "else" which would throw NotImplementedError

Reply:
Done